js/index.js

```diff
@@ -1,42 +1,106 @@
-    "phone" : "1 (888) 888-8888",
-    "email" : "sales@greatidea.io",
-  },
-  "footer": {
-    "copyright" : "Copyright Great Idea! 2018"
-  },
+    "nav": {
+        "nav-item-1": "Services",
+        "nav-item-2": "Product",
+        "nav-item-3": "Vision",
+        "nav-item-4": "Features",
+        "nav-item-5": "About",
+        "nav-item-6": "Contact",
+        "img-src": "img/logo.png"
+    },
+    "cta": {
+        "h1": "DOM Is Awesome",
+        "button": "Get Started",
+        "img-src": "img/header-img.png"
+    },
+    "main-content": {
+        "features-h4": "Features",
+        "features-content": "Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+        "about-h4": "About",
+        "about-content": "About content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+        "middle-img-src": "img/mid-page-accent.jpg",
+        "services-h4": "Services",
+        "services-content": "Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+        "product-h4": "Product",
+        "product-content": "Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+        "vision-h4": "Vision",
+        "vision-content": "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    },
+    "contact": {
+        "contact-h4": "Contact",
+        "address": "123 Way 456 Street Somewhere, USA",
+        "phone": "1 (888) 888-8888",
+        "email": "sales@greatidea.io",
+    },
+    "footer": {
+        "copyright": "Copyright Great Idea! 2018"
+    },
 };
 
 // Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
+let logo = document.querySelector("#logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// NavBar
+let nav = document.querySelectorAll('header nav a');
+nav[0].textContent = siteContent['nav']['nav-item-1']
+nav[1].textContent = siteContent['nav']['nav-item-2']
+nav[2].textContent = siteContent['nav']['nav-item-3']
+nav[3].textContent = siteContent['nav']['nav-item-4']
+nav[4].textContent = siteContent['nav']['nav-item-5']
+nav[5].textContent = siteContent['nav']['nav-item-6']
+
+//Header CTA
+let mainSentence = document.querySelector(".cta .cta-text h1");
+mainSentence.innerHTML = "ANDY <br> IS<br>AWESOME";
```

Comment:
no need for the extra whitespace. it's actually throwing your text off-center if you look closely.
you can do it like this >>
```mainSentence.innerHTML = "ANDY<br>IS<br>AWESOME";```
another thing to note: using .innerHTML has been deprecated due to a security risk. highly advised never to use this method in production code